examples/rules.mk

```diff
@@ -92,7 +92,8 @@ endif
 include $(OPENCM3_DIR)/mk/genlink-config.mk
 endif
 
-SCRIPT_DIR	= $(OPENCM3_DIR)/scripts
+OPENCM3_SCRIPT_DIR = $(OPENCM3_DIR)/scripts
+EXAMPLES_SCRIPT_DIR	= $(OPENCM3_DIR)../scripts
```

Comment:
tried to avoid the double // in the path, but they're all over the file anyway. Updated to double //